packages/flutter/test/painting/continous_rectangle_border_test.dart

```diff
@@ -96,6 +98,86 @@ void main() {
     );
   }, skip: isBrowser);
 
+  testWidgets('Golden test topLeft radii', (WidgetTester tester) async {
+    await tester.pumpWidget(RepaintBoundary(
+      child: Material(
+        color: Colors.green[200],
+        shape: const ContinuousRectangleBorder(
+          borderRadius: BorderRadius.only(
+            topLeft: Radius.elliptical(100.0, 200.0),
+          ),
+        ),
+      ),
+    ));
+
+    await tester.pumpAndSettle();
+
+    await expectLater(
+      find.byType(RepaintBoundary),
+      matchesGoldenFile('continuous_rectangle_border.golden_test_topLeft_radii.png'),
+    );
+  }, skip: isBrowser);
```

Comment:
Do you know why these get skipped in the browser? Maybe add an explaining comment?